app/views/archives/_year_switcher.html.haml

```diff
@@ -0,0 +1,9 @@
+.row.year-switcher
+  .col-xs-6.prev-year
+    = link_to archives_url(year: @year.pred) do
```

Comment:
I thought so, but no: it's `pred` http://ruby-doc.org/core-2.2.2/Integer.html#method-i-pred